source/Engine.h

```diff
@@ -62,6 +62,11 @@ class Weather;
 // situations where there are many objects on screen at once.
 class Engine {
 public:
+	// Minimum allowed spawn period for fleets, used as a
+	// safeguard against bad conditions. This is essentially an
+	// epsilon for comparison to zero.
+	static const int MINIMUM_FLEET_PERIOD = 30;
```

Comment:
I disagree with this, I think only 0 should be considered as 0 spawns (because otherwise it means infinity)

Reply:
I'm changing it to 1, which is equivalent to what you're requesting.

Reply:
I mostly wanted it to be simple, that fixes it on the content side but still its extra lines we dont need, I dont expect this behavior to ever change
if smbd wants to set a spawning to every frame by all means they should be able to

Reply:
It is gone now.